### PR TITLE
Adding namespace filters to K8s integration

### PR DESCRIPTION
**Pull request by alvarocabanas**
Signed-off-by: Alvaro Cabanas <acabanas@newrelic.com>
Co-authored-by: carlossscastro <ccastro@newrelic.com>

This PR adds documentation on configuring namespace filters to our Kubernetes integration.
This feature will be released soon.

### CODE REVIEW

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx [hunk 1]
```diff
@@ -308,6 +308,69 @@ newrelic-infrastructure:
...
+### Filtering Namespaces [#filter-namespace]
+
+The Kubernetes Integration v3 and above allows filtering which namespaces will be scraped by allowing you to include or exclude namespaces based on labels. By default all namespaces will be scraped.
+
+In order to include only namespaces matching a label, change the `namespaceSelector` by adding the following to your `values-newrelic.yaml`, under the `newrelic-infratructure` section:
+
+```yaml
+common:
+  config:
+    namespaceSelector:
```
Comment (paologallinaharbur) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx:320]: Could be just mention that the `namespaceSelector` syntax and behaviour is the K8s one?

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx [hunk 2]
```diff
@@ -308,6 +308,69 @@ newrelic-infrastructure:
...
+newrelic-infrastructure:
+  # ... Other settings as shown above
+  common:
+    config:
+      namespaceSelector:
+        matchExpressions:
+        - {key: newrelic.com/scrape, operator: NotIn, values: ["false"]}
+```
+
 A full list of the settings that can be modified can be found at the [chart's README](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-infrastructure-v3).
```
Comment (paologallinaharbur) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx:374]: this link is broken, can you fix it?

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx (hunk 1 above)
Comment (alvarocabanas) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx:320]: Yes, I'll add it

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx (hunk 2 above)
Comment (alvarocabanas) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx:374]: Yes, going to

Comment (alvarocabanas) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx]: done

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx (hunk 1 above)
Comment (alvarocabanas) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx:320]: done

**Pull request merged**